### ISSUE TITLE
runc lazy-pages test case needs "readonly": false (Error (criu/mount.c:1119): mnt: Can't create a temporary directory: Read-only file system)

### ISSUE THREAD

**Issue by adrianreber** [checkpoint-restore/criu#575]
The test case for lazy-pages in runc still needs:

```
sed -i 's;"readonly": true;"readonly": false;' config.json
```

Without this change I am seeing following error during dumping:
```
# (00.058834) mnt: 	206: 30:/ @ ./sys/fs/cgroup
# (00.058837) mnt: Mount is not fully visible ./sys/fs/cgroup
# (00.058873) mnt: 	mount has children ./sys/fs/cgroup
# (00.058899) Error (criu/mount.c:1119): mnt: Can't create a temporary directory: Read-only file system
# (00.058906) mnt: 	mount is overmounted or has children ./sys/fs/cgroup
# (00.066086) mnt: 	205: 2f:/ @ ./sys
# (00.066099) mnt: 	204: 2a:/ @ ./dev/mqueue
# (00.066132) mnt: 	203: 2e:/ @ ./dev/shm
# (00.069285) mnt: 	202: 2d:/ @ ./dev/pts
# (00.069297) mnt: 	201: 2c:/ @ ./dev
# (00.069300) mnt: Mount is not fully visible ./dev
# (00.069325) mnt: 	mount has children ./dev
# (00.069337) Error (criu/mount.c:1119): mnt: Can't create a temporary directory: Read-only file system
# (00.069341) mnt: 	mount is overmounted or has children ./dev
# (00.076074) mnt: 	200: 2b:/ @ ./proc
# (00.076086) mnt: 	199: 800001:/tmp/busyboxtest/rootfs @ ./
# (00.076106) Dumping file-locks
```

Is this something we can fix in CRIU?

**Comment by avagin** [comment]
Could you try the last version of criu? I think @Snorch fixed this problem.

**Comment by adrianreber** [comment]
Latest criu-dev fails with for the lazy-pages runc test case:
```
# (00.003799) Collecting netns 9/8227
# (00.003801) Switching to 8227's net for collecting sockets
# (00.003857) unix: 	Collected: ino 510676 peer_ino 0 family    1 type    5 state  7 name (null)
# (00.003967) Error (criu/libnetlink.c:27): ERROR -2 reported by netlink (No such file or directory)
# (00.003972) Warn  (criu/sockets.c:791): sockets: The current kernel doesn't support ipv4 raw_diag module
# (00.004061) Error (criu/libnetlink.c:27): ERROR -2 reported by netlink (No such file or directory)
# (00.004063) Warn  (criu/sockets.c:834): sockets: The current kernel doesn't support ipv6 raw_diag module
# (00.004082) netlink: Collect netlink sock 0x7c376
```

**Comment by adrianreber** [comment]
Oh, and I was able to remove the read-only from two test-cases. So I guess some of the fixes for read-only root are working. But this one (read-only: false) , only for the lazy-pages test, is still necessary.

Please also see https://github.com/opencontainers/runc/pull/1936 for the removal of `read-only: false` for two test cases.

**Comment by kolyshkin** [comment]
> # (00.069337) Error (criu/mount.c:1119): mnt: Can't create a temporary directory: Read-only file system

I think it was actually a warning, not an error. Since https://github.com/checkpoint-restore/criu/pull/1033 was merged it is now shown as a warning.

https://github.com/opencontainers/runc/pull/2509 removed the last "readonly: false" setting from the runc checkpoint tests, and it successfully passes a validation, meaning this bug can be closed.

**Comment by kolyshkin** [comment]
This needs to be closed manually since #1033 was merged before I found out it fixes this issue.

Someone who has permissions, please close this one.

**Comment by adrianreber** [comment]
@kolyshkin Thanks.
